content/help/ops/security/repairing-citadel/index.md

```diff
@@ -8,16 +8,17 @@ keywords: [security,citadel,ops]
 If you suspect Citadel isn't working properly, verify the status of the `istio-citadel` pod:
 
 {{< text bash >}}
-$ kubectl get pod -n istio-system
+$ kubectl get pod -l istio=citadel -n istio-system
 NAME                                     READY     STATUS   RESTARTS   AGE
-istio-citadel-ff5696f6f-ht4gq              1/1    Running          0   25d
+istio-citadel-ff5696f6f-ht4gq            1/1       Running  0          25d
 {{< /text >}}
 
-If the `istio-citadel` pod does not exist, you should try to re-run `istio-demo.yaml`.
+If `istio-citadel` pod does not exist, you should try to re-deploy it.
```

Comment:
Keep "the" to make it clear that you are referring to a specific pod, not just any pod. To keep the content easy to translate and to avoid unwarranted complexity, do not employ the conjunctive. I suggest:
If the `istio-citadel` pod doesn't exist, try to re-deploy the pod.